peppermint/1

```diff
@@ -0,0 +1,7 @@
+PRETTY_NAME="PeppermintOS"
+NAME="Peppermint"
+ID=peppermint
+VERSION_CODENAME="bookworm"
```

Comment:
They have a `VERSION_CODENAME` but don't have a `VERSION_ID`

Reply:
rename to `bookworm`?

Reply:
I will enhance the naming convention to cover the edge cases of these 3 PRs

Reply:
I've removed peppermint from this PR.

Reply:
Thanks!